### DISCUSSION TITLE
rpcerror

### DISCUSSION THREAD

**Discussion by yishion819** [immortalwrt/immortalwrt#1187] [Q&A]
![image](https://github.com/immortalwrt/immortalwrt/assets/60944262/08d82cd4-23fb-479a-a9d7-5bad304d0c85)
23.05.1和23.05都会出现这个，我并没有安装皮肤之类的美化，是刷的物理机op，uefi的，不知道如何解决，目前只能重启，隔不了几个小时又会出现，导致提示密码错/无效，无法登录，请大佬赐教，谢谢

**Comment by 1715173329** [comment]
说不定后端崩了，可以用 f12 debug。

**Reply by yishion819** [reply]
谢谢，等下次再出现，我试试
